### DISCUSSION TITLE
Added: LDM Notes for Nov. 28, 2018

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#2033] [General]
[C# Language Design Notes for Nov 28, 2018](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-11-28.md)

1. Are nullable annotations part of array specifiers?
2. Cancellation of async-streams

**Comment by clairernovotny** [comment]
Thank you for adding cancellation to `GetAsyncEnumerator`! ❤️

**Comment by bartdesmet** [comment]
A complete LINQ implementation for `IAsyncEnumerable<T>` in accordance with the latest LDM decisions can be found at https://github.com/dotnet/reactive/pull/423.
